pkgs/top-level/ocaml-packages.nix

```diff
@@ -611,6 +619,8 @@ let
 
     ounit = callPackage ../development/ocaml-modules/ounit { };
 
+    phylogenetics = callPackage ../development/ocaml-modules/phylogenetics { };
+
     pgsolver = callPackage ../development/ocaml-modules/pgsolver { };
```

Comment:
Please preserve the local alphabetical order…

Reply:
Whoops...